src/vm/core_test.zig

```diff
@@ -629,6 +710,166 @@ test "get relocate trace after relocating trace" {
     }
 }
 
```

Comment:
You mentioned this was sourced from running a program in the rust vm, can you clarify which program you ran in a comment in the test?

Reply:
Done